### PR TITLE
[Merged by Bors] - chore(Group/Measure): drop a `T2Space` assumption

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by sgouezel** [comment]
bors r+
Thanks!

**Review by grunweg — COMMENTED** [review]
Nice, thanks!
maintainer merge

**Comment by sgouezel** [comment]
bors r-
Can you fix the lint, and then send it back to bors?
bors d+

**Comment by mattrobball** [comment]
bors r- 
bors delegate+

**Comment by mattrobball** [comment]
Oh odd. I just got a new bors failure alert for a batch with this

File: Mathlib/MeasureTheory/Measure/Haar/NormedSpace.lean [hunk 1]
```diff
@@ -30,23 +30,19 @@ example {E : Type*} [NormedAddCommGroup E] [NormedSpace ℝ E] [Nontrivial E] [F
...
-
-variable [CompleteSpace 𝕜] [T2Space G] [FiniteDimensional 𝕜 G] [ContinuousSMul 𝕜 G]
-  [ContinuousSMul 𝕜 H]
+  [IsAddHaarMeasure μ] [BorelSpace G] [BorelSpace H]
+  [CompleteSpace 𝕜] [T2Space G] [FiniteDimensional 𝕜 G] [ContinuousSMul 𝕜 G]
+  [ContinuousSMul 𝕜 H] [T2Space H]
 
 instance MapLinearEquiv.isAddHaarMeasure (e : G ≃ₗ[𝕜] H) : IsAddHaarMeasure (μ.map e) :=
-  MapContinuousLinearEquiv.isAddHaarMeasure _ e.toContinuousLinearEquiv
+  e.toContinuousLinearEquiv.isAddHaarMeasure_map _
```
Comment (urkud) [inline Mathlib/MeasureTheory/Measure/Haar/NormedSpace.lean:43]:
```suggestion
  e.toContinuousLinearEquiv.isAddHaarMeasure_map _
```

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**